src/js/background.js

```diff
@@ -776,6 +780,25 @@ function startBackgroundListeners() {
     }
   }, {urls: ["http://*/*", "https://*/*"]}, []);
 
+  // Temporary fix for android while it doesn't support `browser_action.default_popup`
+  // Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1330159
+  if(!chrome.browserAction.getPopup) {
+    chrome.browserAction.onClicked.addListener(() => {
+      chrome.tabs.query({active: true, lastFocusedWindow: true}, (tabs) => {
+        var url = chrome.runtime.getManifest().browser_action.default_popup + "?id=" + tabs[0].id;
+
+        // check if popup already is open
+        chrome.tabs.query({url}, (popupTabs) => {
+          if(popupTabs.length > 0){
+            chrome.tabs.update(popupTabs[0].id, {active: true});
+            chrome.tabs.reload(popupTabs[0].id);
+          } else {
+            chrome.tabs.create({url, index: tabs[0].index + 1});
```

Comment:
I think that option 2 is the best! It also mimics the behavior of current popups (it isn't long running).  And adding code for deactivating the popup should be simpler than adding the updating code for 3. And I think it will avoid the confusion a user might have when they switch to a popup, and don't know what tab the popup is for.